ansible/roles_ocp_workloads/ocp4_workload_cyberark_dap/tasks/remove_workload.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Implement your workload removal tasks here
+# ------------------------------------------
+
+- name: Remove users read-only access to the DAP config map
+  k8s:
+    state: absent
+    definition:
+      kind: RoleBinding
+      apiVersion: rbac.authorization.k8s.io/v1
+      metadata:
+        name: dap-cm-access-user{{ item }}
+      roleRef:
+        kind: ClusterRole
+        name: dap-cm-access-role
+        apiGroup: rbac.authorization.k8s.io
+      subjects:
+      - kind: User
+        name: user{{ item }}
+  loop: "{{ range(1, num_users | int + 1, 1) | list }}"
+
+- name: Remove lab user demo projects
+  k8s:
+    state: absent
+    definition:
+      apiVersion: project.openshift.io/v1
+      kind: Project
+      metadata:
+        name: 'user{{ item }}'
+  loop: "{{ range(1, num_users | int + 1, 1) | list }}"
+
+- name: Remove MySQL
+  k8s:
+    state: absent
+    src: /home/{{ansible_user}}/cyberark_setup/mysql.yaml
+
+- name: Remove CyberArk Master
+  k8s:
+    state: absent
+    src: /home/{{ansible_user}}/cyberark_setup/master-deployment-manifest.yaml
+
+- name: Remove CyberArk resources
+  k8s:
+    state: absent
+    definition: "{{ lookup('template', item ) | from_yaml }}"
+  loop:
+  - ./templates/namespace.j2
+  - ./templates/service.j2
+  - ./templates/dap_authn_role.j2
+  - ./templates/role_binding.j2
+  - ./templates/dap_cm_role.j2
+  - ./templates/secrets_cluster_role.j2
+
+- name: Remove cyberark setup files
+  become: true
+  file:
+    state: absent
+    path: "{{ item }}"
+  loop:
+  - /home/{{ansible_user}}/cyberark_setup
```

Comment:
No need to use `loop` with only a single item. Also, please use `~{{ ansible_user}}/cyberark_setup` rather than `/home/` so as to avoid issues if the home directory is not under `/home/`.

Reply:
Done, thank you!